go.mod

```diff
@@ -2,20 +2,23 @@ module github.com/crossplane/terrajet
 
 go 1.17
 
+// Due to https://github.com/crossplane/terrajet/issues/294
+replace github.com/hashicorp/go-getter => github.com/hashicorp/go-getter v1.6.2
```

Comment:
do we have a replace directive because we do not directly depend on this package? 🤔